packages/peregrine/lib/talons/SearchPage/useSearchPage.js

```diff
@@ -113,7 +116,8 @@ export const useSearchPage = props => {
                 currentPage: Number(currentPage),
                 filters: newFilters,
                 inputText,
-                pageSize: Number(PAGE_SIZE)
+                pageSize: Number(PAGE_SIZE),
+                sort: { [String(sortAttribute)]: String(sortDirection) }
```

Comment:
I see that you did this in the category implementation, but casting is usually a smell, unless that data type is out of your control (ie. data fetch). If you can guarantee this state will always be a string, I would consider removing this.